inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -62,4 +62,17 @@ public function set_option_on_update( $old_version ) {
 
 		update_option( 'wp_rocket_settings', $options );
 	}
+
+	/**
+	 * Determines if Remove Unused CSS option is enabled.
+	 *
+	 * @return boolean
+	 */
+	public function is_allowed() : bool {
+		if ( ! (bool) $this->options->get( 'remove_unused_css', 0 ) ) {
+			return false;
+		}
+
+		return true;
+	}
```

Comment:
```suggestion
	public function is_enabled() : bool {
		return (bool) $this->options->get( 'remove_unused_css', 0 ) );
	}
```

2 suggestions:
- name the method `is_enabled()` to better reflect it's related to the option being enabled specifically
- simplify the method code